lib/hotpatch-rebalancer-agent/hotpatcher.js

```diff
@@ -247,7 +249,17 @@ Hotpatcher.prototype.listStorInsts = function listStorInsts(cb) {
                             if (err) {
                                 next(err);
                             } else {
-                                storInsts = insts;
+                                // `insts` is the set of storage instances in
```

Comment:
This section is the actual fix. The "completion" parts of just a minor doc buglet.